test/support/factory.ex

```diff
@@ -62,7 +62,9 @@ defmodule Livebook.Factory do
       mode: "offline",
       clustering: "",
       zta_key: "",
-      zta_provider: :""
+      zta_provider: :"",
```

Comment:
Nit: these two should be nullable in schema and then we can use nil as defaults.

Reply:
The `TeamClient` converts to atom, but I'll change to make it nullable